src/checkpoint/dispatch/vrt/object_registry.h

```diff
@@ -132,12 +132,12 @@ inline auto getObjSize(TypeIdx han) {
 
 template <typename T>
 inline auto getObjAllocate(TypeIdx han) {
```

Comment:
These now ought not to be named `getObjFoo`

How about `allocateConcreteType` and `constructConcreteType`?

Reply:
done